lib/install/management/delete.go

```diff
@@ -188,9 +190,12 @@ func (d *Dispatcher) detachAttachedDisks(v *vm.VirtualMachine) error {
 	return err
 }
 
-func (d *Dispatcher) DeleteVCHInstances(vmm *vm.VirtualMachine, conf *config.VirtualContainerHostConfigSpec) error {
+func (d *Dispatcher) DeleteVCHInstances(vmm *vm.VirtualMachine, conf *config.VirtualContainerHostConfigSpec, containers *string) error {
 	defer trace.End(trace.Begin(conf.Name))
 
+	deletePoweredOnContainers := d.force || (containers != nil && *containers == "all")
```

Comment:
Where is `"all"` coming from? We should consider making it a constant since we're using it in two places here and it may be referenced in other places in the future.

Reply:
Good catch! `"all"` is defined in Swagger, and is captured in a generated constant.

I could use that here, but that seems conceptually wrong (since this code shouldn't "know about" swagger).

I'll look for a better solution.

Reply:
What do you think of the approach in b5b48d08? I don't love it, but it avoids "magic constants", provides type safety, avoids coupling the management layer to swagger, and is easily extended if we introduce more granular deletion specifications.

Reply:
Looks great - thanks!